src/test/java/com/teragrep/cfe_35/router/OutputFailureTest.java

```diff
@@ -99,37 +101,44 @@ private void setupRouter() {
     }
 
     private final List<byte[]> spoolList = new ArrayList<>();
-    private Server spoolServer;
+    private EventLoop spoolEventLoop;
     private final List<byte[]> inspectionList = new ArrayList<>();
-    private Server inspectionServer;
+    private EventLoop inspectionEventLoop;
 
     public void setupTargets() throws IOException {
-        spoolServer = setup(7601, spoolList);
-        inspectionServer = setup(7602, inspectionList);
+        spoolEventLoop = setup(7601, spoolList);
+        inspectionEventLoop = setup(7602, inspectionList);
     }
 
-    private Server setup(int port, List<byte[]> recordList) throws IOException {
+    private EventLoop setup(int port, List<byte[]> recordList) throws IOException {
         Consumer<FrameContext> cbFunction = relpFrameServerRX -> recordList
                 .add(relpFrameServerRX.relpFrame().payload().toBytes());
 
-        ExecutorService executorService = Executors.newSingleThreadExecutor();
+        EventLoopFactory eventLoopFactory = new EventLoopFactory();
+        EventLoop eventLoop = eventLoopFactory.create(); // FIXME this is not cleaned up
+        Thread eventLoopThread = new Thread(eventLoop);
+        eventLoopThread.start(); // FIXME this is not cleaned up
+
+        ExecutorService executorService = Executors.newSingleThreadExecutor(); // FIXME this is not cleaned up
+
         ServerFactory serverFactory = new ServerFactory(
+                eventLoop,
                 executorService,
                 new PlainFactory(),
-                () -> new DefaultFrameDelegate(cbFunction)
+                new FrameDelegationClockFactory(() -> new DefaultFrameDelegate(cbFunction))
         );
-        Server server = serverFactory.create(port);
-        Thread serverThread = new Thread(server);
-        serverThread.start();
-        return server;
+        serverFactory.create(port);
+
+        return eventLoop;
     }
 
     private void teardownTargets() throws InterruptedException {
-        spoolServer.stop();
-        inspectionServer.stop();
+        spoolEventLoop.stop();
+        inspectionEventLoop.stop();
     }
 
     @Test
+    @Disabled
```

Comment:
Add a reason why it is disabled, or make it toggleable using Enabled/Disabled IfSystemProperty calls